packages/ui/src/composables/useModalLevel.ts

```diff
@@ -0,0 +1,27 @@
+import { computed, ref } from 'vue'
+
+const LOWEST_MODAL_LEVEL = 1
+const NO_MODALS_LEVEL = 0
+const currentModalsLevel = ref(NO_MODALS_LEVEL)
+
+export const useModalLevel = () => {
+  const currentModalLevel = ref<null | number>(null)
+  const getModalLevelOnClose = () => {
+    currentModalsLevel.value--
+    currentModalLevel.value = null
+  }
+  const getModalLevelOnOpen = () => {
+    currentModalsLevel.value++
+    currentModalLevel.value = currentModalsLevel.value
```

Comment:
> `Will Will Smith smith? Yes, Will Smith will smith`

How about changing `currentModalsLevel` to `theUpperModalLevel`?

Reply:
Changed